dvc/repo/add.py

```diff
@@ -11,6 +27,8 @@ def add(repo, fname, recursive=False, no_commit=False):
 
     for stage in stages:
         stage.dump()
+    for stage in stages:
+        repo.scm.ignore_remove(stage.path)
```

Comment:
Why are we removing stage.path from gitignore at all?

Reply:
Sorry, leftover after initial changes